HLTrigger/Configuration/test/OnLine_HLT_PIon.py

```diff
@@ -3063,9 +3063,6 @@
   useMagVolumes = cms.bool( True ),
   returnTangentPlane = cms.bool( True )
 )
-process.TrackerAdditionalParametersPerDetESModule = cms.ESProducer( "TrackerAdditionalParametersPerDetESModule",
-  appendToDataLabel = cms.string( "" )
-)
```

Comment:
Please remove the changes from HLTrigger/Configuration/test/OnLine_HLT_PIon.py and restore th eoriginal: this file is generated and the next iteration will overwrite any changes in here. Any changes should be done in HLTrigger/Configuration/python/customizeHLTforCMSSW.py